packages/ember-data/tests/integration/store-test.js

```diff
@@ -1,3 +1,5 @@
+import { badIdFormatAssertion } from 'ember-data/system/store';
```

Comment:
I think this is a limitation of our testing infrastructure. Modules can be access in dev mode but the production build (which I suspect travis is using) wraps Ember Data's module loader in a closure so tests (or user code) can't get at internal paths like 'ember-data/system/store'.